docs/containers.conf.5.md

```diff
@@ -422,6 +422,25 @@ Name of destination for accessing the Podman service. See SERVICE DESTINATION TA
 The cgroup management implementation used for the runtime. Supports `cgroupfs`
 and `systemd`.
 
+**compat_api_enforce_docker_hub**=true
+
+Enforce using docker.io for completing short names in Podman's compatibility
+REST API. Note that this will ignore unqualified-search-registries and
+short-name aliases defined in containers-registries.conf(5).
+
+**compose_providers**=[]
+
+Specify one or more external providers for the compose command.  The first
+found provider is used for execution. Can be an absolute path or a (file) name.
+Can be an absolute and relative path or a (file) name.
```

Comment:
The two sentences look similar. Maybe one should be removed?

The same thing appear in the file
pkg/config/containers.conf

Reply:
Thanks for catching!